### PR TITLE
Add Release Delay

### PR DESCRIPTION
**Pull request by joschi36**

### CODE REVIEW

**Comment by onitake** [comment]
[Brillant!](https://thedailywtf.com/articles/The_Brillant_Paula_Bean)


(and no, I didn't mean it ironically - i think the delay feature is genuinely good)

File: cmd/kured/main.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func rebootBlockingPodsExist(client *kubernetes.Clientset, nodeID string) bool {
 func releaseBlockingPodsNotready(client *kubernetes.Clientset, nodeID string) bool {
 	fieldSelector := fmt.Sprintf("spec.nodeName=%s", nodeID)
 	for _, labelSelector := range releasePodSelectors {
-		podList, err := client.CoreV1().Pods("").List(metav1.ListOptions{
+		podList, err := client.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{
```
Comment (onitake) [inline cmd/kured/main.go:213]: Are you sure context.TODO is appropriate here?
Normally a context object is passed through all layers, so state can propagate. TODO is mostly a placeholder.
Do you intend to make releaseBlockingPodsNotready accept a context argument?

Comment (onitake) [reply cmd/kured/main.go:213]: context.Background may be more appropriate if that's not the case.

Comment (joschi36) [reply cmd/kured/main.go:213]: They do it [here](https://github.com/swisstxt/kured/blob/ddef56b67478fb7d032a59fd8b9dcfcf3af25a11/cmd/kured/main.go#L184), so I thought I do it the same here too. :joy: 

Actually I have no plan what I'm doing, but as far as I can tell it works

Comment (onitake) [reply cmd/kured/main.go:213]: Do you know where these changes are coming from?
I'll open an issue on their bug tracker, and we can close the discussion here.

Comment (joschi36) [reply cmd/kured/main.go:213]: https://github.com/weaveworks/kured/commit/16109017cec02a8a0c4f80f829acc38ab28a14b2

Comment (onitake) [reply cmd/kured/main.go:213]: Ref: https://github.com/weaveworks/kured/issues/234

**Comment by joschi36** [comment]
Merged manually: https://github.com/swisstxt/kured/commit/e12a7aaeac8cedf0efa4565b43b772cb460d03d9

**Pull request closed without merging**